projects/demo/src/main.browser.ts

```diff
@@ -9,7 +9,47 @@ if (environment.production) {
 }
 
 const bootstrap = async (): Promise<NgModuleRef<AppBrowserModule>> =>
-    platformBrowserDynamic().bootstrapModule(AppBrowserModule);
+    platformBrowserDynamic().bootstrapModule(AppBrowserModule, {
+        /**
+         * Optionally specify coalescing event change detections or not.
+         * Consider the following case.
+         *
+         * <div (click)="doSomething()">
+         *   <button (click)="doSomethingElse()"></button>
+         * </div>
+         *
+         * When button is clicked, because of the event bubbling, both
+         * event handlers will be called and 2 change detections will be
+         * triggered. We can colesce such kind of events to only trigger
+         * change detection only once.
+         *
+         * By default, this option will be false. So the events will not be
+         * coalesced and the change detection will be triggered multiple times.
+         * And if this option be set to true, the change detection will be
+         * triggered async by scheduling a animation frame. So in the case above,
+         * the change detection will only be triggered once.
+         */
+        ngZoneEventCoalescing: true,
```

Comment:
We probably need to be able to start our portal in different modes to test, like we have modes for default change detection and shadow dom. Since it defers change detection to animationFrame it can drastically change behavior and we need to support both modes.

Reply:
https://github.com/Tinkoff/taiga-ui/pull/3535